### PR TITLE
Proposal Detail : Features migrated from conviction-voting

### PR DESCRIPTION
**Pull request by rperez89**
This PR include the latest changes to the proposal screen on the conviction voting repo

### CODE REVIEW

File: src/hooks/useProposalActions.js [hunk 1]
```diff
@@ -29,6 +29,7 @@ export default function useProposalActions(onDone) {
 
   const stakeToProposal = useCallback(
     (proposalId, amount) => {
+      console.log('STAKE ', typeof amount)
```
Comment (fabriziovigevani) [inline src/hooks/useProposalActions.js:32]:
```suggestion

```

File: src/hooks/useProposalActions.js [hunk 2]
```diff
@@ -87,6 +88,10 @@ async function sendIntent(
 
     const txPath = await intent.paths(from)
 
+    console.log('txPath ', txPath)
+
+    console.log('PARAMS! ', params)
+
```
Comment (fabriziovigevani) [inline src/hooks/useProposalActions.js:94]:
```suggestion

```

File: src/screens/ProposalDetail.js [hunk 3]
```diff
@@ -40,7 +53,7 @@ function ProposalDetail({
 
   const { account: connectedAccount } = useWallet()
   const chartLines = useConvictionHistory(proposal)
-
+  const { stakeToken, accountBalance } = useAppState()
```
Comment (fabriziovigevani) [inline src/screens/ProposalDetail.js:56]: Ah i think in this case the `stakeToken` is already passed from `App.js`

Comment (rperez89) [reply src/screens/ProposalDetail.js:56]: i think that the one passed is re requestToken

**Pull request merged**